MonstieWash/Assets/Scripts/Consumables/ConsumablesManager.cs

```diff
@@ -15,10 +15,10 @@ public class ConsumablesManager : MonoBehaviour
     [Tooltip("Gameobject new Ui consumable items will be parented under")] [SerializeField] private GameObject imageHolder; //Empty Gameobject in Ui to hold images; (I tried changing to transform as per feedback but doesn't seem to work with rect transform in ui)
     [Tooltip("A template obj requiring, image, UIconsumableScript, button linked to the scripts onclick")] [SerializeField] private GameObject refObject; //blank object for instantiate to reference; Needs a button, UIConsumableScript, Image.
     [Tooltip("Layer that checks player mouse is over monster collider")] [SerializeField] private LayerMask monsterLayer; //Layer of monster hitbox;
-    [Tooltip("Layer that checks player mouse is over monster collider")] [SerializeField] public LayerMask consumableLayer; //Layer of consumables hitbox;
+    [Tooltip("Layer that checks player mouse is over bag to open its contents")] [SerializeField] public LayerMask bagLayer; //Layer of consumables hitbox;
+    [Tooltip("Layer that checks player mouse is over consumable collider")] [SerializeField] public LayerMask itemConsumableLayer; //Layer of consumables hitbox;
```

Comment:
very minor, but these should be marked as private since they are not meant to be modified from outside the class. If they are read from outside the class (as in the case of `itemConsumableLayer`), a public getter field should be created.

```suggestion
    [Tooltip("Layer that checks player mouse is over bag to open its contents")] [SerializeField] private LayerMask bagLayer; //Layer of consumables hitbox;
    [Tooltip("Layer that checks player mouse is over consumable collider")] [SerializeField] private LayerMask itemConsumableLayer; //Layer of consumables hitbox;
    
    public ItemConsumableLayer { get { return itemConsumableLayer; } }
```